src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -53,3 +53,13 @@
     type: command
     short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
 """
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace usage data by an Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for an Azure subscription within a billing period.
```

Comment:
Not sure about the wording here. Maybe `Show marketplace billing information for...` ?

Reply:
Marketplace and billing period are two different concepts. Here is the marketplace within a billing period.